Aspire Budgeting/Facilities/LocalAuthorizationManager.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  LocalAuthorizationManager.swift
+//  Aspire Budgeting
+//
+//  Created by TeraMo Labs on 11/7/19.
+//  Copyright © 2019 TeraMo Labs. All rights reserved.
+//
+
+import Foundation
+import LocalAuthentication
+
+class LocalAuthorizationManager: ObservableObject {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mohitathwani/aspirebudgeting/blob/9f83f9ce5d382a60d8a983f7fa5efc14996a2311/Aspire Budgeting/Facilities/LocalAuthorizationManager.swift#L12"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>